io.github.lavenderdotpet.LibreQuake.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.lavenderdotpet.LibreQuake
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: librequake.sh
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+
+modules:
+  - shared-modules/libmad/libmad.json
+
+  - name: quakespasm
+    sources:
+      - type: git
+        url: https://github.com/sezero/quakespasm.git
+        tag: quakespasm-0.96.3
+    buildsystem: simple
+    build-commands:
+      - make --directory=Quake USE_SDL2=1 LDFLAGS="-L/app/lib"
+      - install -D -m755 -t /app/bin/ Quake/quakespasm
+      - install -D -m644 -t /app/bin/ Quake/quakespasm.pak
+
+  - name: librequake-data
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: "https://github.com/lavenderdotpet/LibreQuake/releases/download/v0.07-beta/full.zip"
+        sha256: "59dd151c2556ae95d92183053683cc842485fa49265a7cacf6566e0f500072d3"
+    build-commands:
+      - install -d /app/share/librequake
+      - mv id1 /app/share/librequake/
+
+  - name: launch-script
+    sources:
+      - type: script
+        dest-filename: librequake.sh
+        commands:
+          - cp -avu /app/share/librequake/id1 "$XDG_DATA_HOME"/
```

Comment:
So this will get executed every time it's run.

Why is this needed. This seems weird.

Reply:
I have found a cleaner solution for this. This was here to persist userdata (config, screenshots etc.). Quakespasm kind of supports user directories natively so I used this now.